requery/src/main/java/io/requery/converter/CurrencyConverter.java

```diff
@@ -0,0 +1,55 @@
+package io.requery.converter;
+
+import java.util.Currency;
+
+import javax.annotation.Nullable;
+
+import io.requery.Converter;
+
+/**
+ * A {@link Converter} between a {@link Currency} and its ISO 4217 code.
+ */
+public class CurrencyConverter implements Converter<Currency, String> {
+
+    /**
+     * The maximum length of the ISO 4217 code of available currencies (determines the size of
+     * database field).
+     */
+    public static final int PERSISTED_SIZE = 3;
+
+// Commented because it requires API level 14
```

Comment:
You can remove commented out code, can always refer back to the PR if the min api version changes